diff --git a/node_modules/5to6-codemod/transforms/exports.js b/node_modules/5to6-codemod/transforms/exports.js
index 55f2a19..fe4fd22 100644
--- a/node_modules/5to6-codemod/transforms/exports.js
+++ b/node_modules/5to6-codemod/transforms/exports.js
@@ -44,10 +44,10 @@ module.exports = function(file, api, options) {
 		})
 		filteredPaths.forEach(function (p, i) {
 			// aggregate all specifiers
-			specifiers.push(j.exportSpecifier(
-				j.identifier(p.value.right.name),
-				j.identifier(p.value.right.name)
-			))
+			specifiers.push(j.exportSpecifier.from({
+			  exported: j.identifier(p.value.right.name),
+			  local: j.identifier(p.value.right.name),
+			}));
 
 			// replace the last module.exports.*
 			if (i === filteredPaths.length - 1) {
